influxdb_iox/tests/end_to_end_cases/cli.rs

```diff
@@ -196,6 +189,132 @@ async fn remote_partition_and_get_from_store_and_pull() {
+                        .arg("partition")
+                        .arg("show")
+                        .arg("1")
+                        .assert()
+                        .success()
+                        .get_output()
+                        .stdout
+                        .clone();
+
+                    let object_store_id = get_object_store_id(&out);
+                    let dir = tempdir().unwrap();
+                    let f = dir.path().join("tmp.parquet");
+                    let filename = f.as_os_str().to_str().unwrap();
+
+                    Command::cargo_bin("influxdb_iox")
+                        .unwrap()
+                        .arg("-h")
+                        .arg(&router_addr)
+                        .arg("remote")
+                        .arg("store")
+                        .arg("get")
+                        .arg(&object_store_id)
+                        .arg(filename)
+                        .assert()
+                        .success()
+                        .stdout(
+                            predicate::str::contains("wrote")
+                                .and(predicate::str::contains(filename)),
+                        );
+
+                    // convert to line protocol (stdout)
+                    let output = Command::cargo_bin("influxdb_iox")
+                        .unwrap()
+                        .arg("debug")
+                        .arg("parquet-to-lp")
+                        .arg(filename)
+                        .assert()
+                        .success()
+                        .stdout(predicate::str::contains(line_protocol))
+                        .get_output()
+                        .stdout
+                        .clone();
+
+                    println!("Got output  {:?}", output);
+
+                    // test writing to output file as well
+                    // Ensure files are actually wrote to the filesystem
+                    let output_file =
+                        tempfile::NamedTempFile::new().expect("Error making temp file");
+                    println!("Writing to  {:?}", output_file);
+
+                    // convert to line protocol (to a file)
+                    Command::cargo_bin("influxdb_iox")
+                        .unwrap()
+                        .arg("debug")
+                        .arg("parquet-to-lp")
+                        .arg(filename)
+                        .arg("--output")
+                        .arg(output_file.path())
+                        .assert()
+                        .success();
+
+                    let file_contents =
+                        std::fs::read(output_file.path()).expect("can not read data from tempfile");
+                    let file_contents = String::from_utf8_lossy(&file_contents);
+                    assert!(
+                        predicate::str::contains(line_protocol).eval(&file_contents),
+                        "Could not file {} in {}",
```

Comment:
I think this is supposed to say:

```suggestion
                        "Could not find {} in {}",
```